batch7crud-roll2/src/main/java/com/lftechnology/batch7crud/service/EmployeeService.java

```diff
@@ -28,8 +38,15 @@ public Employee fetchById(int id) throws DataException {
     return employeeDao.fetchById(id);
   }
 
-  public void update(Employee employee) throws DataException {
-    employeeDao.update(employee);
+  public void update(Employee employee) throws DataException, ValidationException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: !unknownSymbol!, !unknownSymbol! [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1160)